dask/tests/test_order.py

```diff
@@ -374,7 +374,7 @@ def _(*args):
     }
 
     order(dsk)
-    dask.get(dsk, [a1, b1, c1])  # trigger computation
+    dask.local.get_sync(dsk, [a1, b1, c1])  # trigger computation
```

Comment:
This test seems to be sensitive to scheduler behavior (even the synchronous one where tasks are just run in order received). There may be an orthogonal bug here.

Reply:
It appears the test here was just wrong. Have fixed it. More details in comment ( https://github.com/dask/dask/pull/6322#discussion_r597915971 )